README.rst

```diff
@@ -19,6 +19,9 @@ dwave-neal
 .. index-start-marker
 
 An implementation of a simulated annealing sampler.
+A simulated annealing sampler can be used for approximate Boltzmann sampling, or heuristic optimization. In this implementation the equilibrium distribution at a target inverse temperature (beta) is approached by performing updates at a sequence of increasing beta values (the beta_schedule), terminating at the target beta. In this implementation each spin is updated once in a fixed order per point in the beta_schedule according to a Metropolis-Hastings update. When beta is large the target distribution concentrates, at equilibrium, over ground states of the model. Samples are guaranteed to match the equilibrium for long 'smooth' beta_schedules, in practical operation the sampler can be used as a heuristic for equilibrium sampling, and optimization.
```

Comment:
Add an empty line after the first sentence. Break the new paragraph into lines of up to 80 chars.